.coafile

```diff
@@ -0,0 +1,37 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, .eslintrc.js, tmp/**
+use_spaces = true
+
```

Comment:
Can we just add `SpaceConsistencyBear` here to avoid repetition?

Reply:
or just add a section called space and it should contain all files.

A section doesn't have to be called by their file extension, and it shouldn't be if it causes a lot of repetition.

Reply:
I use it that way for performance reason.

Reply:
It is faster this way. instead of per-bear basis.

Reply:
Hmm I'm not sure if we should sacrifice for it @jayvdb ?

Reply:
What is being sacrificed? Am i missing something again?

Reply:
consistency and maintainability is being sacrificed, ... and what for?  a few milliseconds of build time.  Have you done performance testing on this?

We will be generating these .coafiles soon, from moban templates.  https://gitlab.com/coala/mobans/issues/1

Having generic rules and specific overrides where absolutely necessary improves our ability to understand how each repo is different, and manage large numbers of repositories in a consistent manner.

Reply:
Not a few milliseconds in my I5 laptop processor, it is ~10 seconds.

Reply:
It was when it only have 3 rules. Now it have 9 Rules.

Reply:
I though i open discussion i found posible performance improvement somewhere in coala gitter. Can't found it.